lepton/openstack.go

```diff
@@ -354,19 +339,52 @@ func (o *OpenStack) ListInstances(ctx *Context) error {
 				}
 			}
 
-			row = append(row, s.Name)
-			row = append(row, ipv4)
-			row = append(row, s.Status)
-			row = append(row, s.Created.Format("2006-01-02 15:04:05"))
-
-			table.Append(row)
+			cinstance := CloudInstance{
+				Name:      s.Name,
+				PublicIps: []string{ipv4},
+				Status:    s.Status,
+				Created:   s.Created.Format("2006-01-02 15:04:05"),
+			}
 
+			cinstances = append(cinstances, cinstance)
 		}
-		table.Render()
 
-		return false, nil
+		return true, nil
```

Comment:
Replaced by true to iterate over all pages. The value false makes the iterator end prematurely.

@eyberg confirm whether reading only the first page is intentional.

Reply:
nope - just a bug - thanks!